src/graphorm/index.ts

```diff
@@ -143,6 +144,15 @@ const obj = new GraphORM({
       public: {
         transform: (value: boolean): boolean => !value,
       },
+      permissions: {
+        transform: (_, ctx: Context, source: GQLSource) => {
+          if (!ctx.userId || !source.currentMember) {
```

Comment:
We discussed that this field should be in `SourceMember` not in `Source` please adjust